fs/write_json.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as path from "./path/mod.ts";
+
+interface WriteJsonOption {
+  spaces?: number | string;
+  replacer?: (number | string)[] | null | Replacer;
+}
+
+type Replacer = (key: string, value: any) => any;
+
+/**
+ * Writes an object to a JSON file.
+ * @export
+ * @param {string} filePath
+ * @param {*} object
+ * @param {WriteJsonOption} [options]
+ * @returns {Promise<void>}
```

Comment:
Aren't these docs superfluous given type annotations? CC @kitsonk

Reply:
Correct.  We shouldn't be using JSDoc like this.

Reply:
Without revelant descriptions of parameters?

Reply:
If you want to provide descriptions you can do that, this JSDoc is not doing that.  If providing descriptions is informative then the following would be acceptable:

```ts
/** Some description
 * @param filePath The path to the file
 */
```

What is included here is not acceptable.  Need to find out if there is an eslint rule for this.

Reply:
you mean this one: https://eslint.org/docs/rules/valid-jsdoc ?

Reply:
No, because those include the types... we want something that does not include the types.

@j-f1 any rule you know of to keep types out of JSDoc?

Reply:
@kitsonk you may look this option: https://eslint.org/docs/rules/valid-jsdoc#requireparamtype . If set to false it fits your needs.

Reply:
https://github.com/gajus/eslint-plugin-jsdoc/issues/134

Reply:
Oh right. But adding enforce of JsDoc good atm? And when this feature will be integrated we add this config to eslint no?

Reply:
I think this interface is obvious and there is no need to explain the parameters.
So I only leave a single line comment